### ISSUE TITLE
itest-flake: on chain fund recovery bitcoind, utxo check race cond

### ISSUE THREAD

**Issue by Roasbeef** [lightningnetwork/lnd#5532]
Unclear if something earlier in the test tripped things up, the assertion failing is wrapped in a `wait.Predicate` already: 
```
=== RUN   TestLightningNetworkDaemon/04-of-82/bitcoind/onchain_fund_recovery
    test_harness.go:88: Failed: (onchain fund recovery): exited with error: 
        *errors.errorString expected restored node to have 200000000 satoshis, instead has 0 satoshis, expected 2 utxos instead has 0
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_recovery_test.go:187 (0x14426e5)
        	testOnchainFundRecovery.func1: t.Fatalf("expected restored node to have %d satoshis, "+
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_recovery_test.go:267 (0x13fab9c)
        	testOnchainFundRecovery: restoreCheckBalance(2*btcutil.SatoshiPerBitcoin, 2, 1, skipAndSend(9))
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/test_harness.go:112 (0x13a7dee)
        	(*harnessTest).RunTestCase: testCase.test(h.lndHarness, h)
        /home/travis/gopath/src/github.com/lightningnetwork/lnd/lntest/itest/lnd_test.go:247 (0x144b32f)
        	TestLightningNetworkDaemon.func4: ht.RunTestCase(testCase)
        /home/travis/.gimme/versions/go1.16.3.linux.amd64/src/testing/testing.go:1193 (0x52232f)
        	tRunner: fn(t)
        /home/travis/.gimme/versions/go1.16.3.linux.amd64/src/runtime/asm_amd64.s:1371 (0x472001)
        	goexit: BYTE	$0x90	// NOP
=== CONT  TestLightningNetworkDaemon
    lnd_test.go:255: Failure time: 2021-07-16 18:26:47.522
--- FAIL: TestLightningNetworkDaemon (966.79s)
```

**Comment by yyforyongyu** [comment]
Probably fixed by #5637

**Comment by bhandras** [comment]
I see similar flake quite a bit on ARM lately. It looks like that Travis ARM performance has degraded a lot lately and 30s default is simply not enough. Also we're polling walletbalance and listunspent quite a bit so that might also be an issue. See eg: https://termbin.com/ytdy

How about just chaning the timeout to something longer? https://github.com/lightningnetwork/lnd/pull/5688

**Comment by yyforyongyu** [comment]
Same test with different flake #5494. Now that we require the node to be fully started (synced), these flakes should be gone. Will keep an eye on them.

> It looks like that Travis ARM performance has degraded a lot lately and 30s default is simply not enough. 

Yeah I've noticed that too. There's a new param, [`NodeStartTimeout`](https://github.com/lightningnetwork/lnd/blob/master/lntest/timeouts_darwin.go#L31), which is set to 120s for ARM. This should help the situation a bit.

**Comment by yyforyongyu** [comment]
fixed by #6825
